Clients/src/presentation/pages/EntityGraph/utils/gapDetector.ts

```
@@ -0,0 +1,164 @@
+  // Check risks
+  data.risks.forEach(risk => {
+    const gaps: GapResult['gaps'] = [];
+    const riskRules = enabledRules.filter(r => r.entityType === 'risk');
+    const daysSince = getDaysSinceCreation(risk.created_at);
+
+    riskRules.forEach(rule => {
+      let hasProblem = false;
+      switch (rule.requirement) {
+        case 'has_control':
+          // Risks are mitigated by controls in their project
+          const projectId = risk.project_id || (risk.model_id ? data.models.find(m => m.id === risk.model_id)?.projects?.[0] : undefined);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)